README.md

```diff
@@ -1,61 +1,2354 @@
-# <Name of Topic> Masterclass
+# Docker Masterclass
 
-* **Masterclass Type:** <Core/Deepdive>
-* **Maximum Expected Time To Complete:** <Time in minutes>
+* **Masterclass Type:** Core
+* **Maximum Expected Time To Complete:** 2 hours
 
-[Optional Depending on Prerequisites]
-## Prerequisite Classes
+## Table of Contents
+- [Docker Masterclass](#docker-masterclass)
```

Comment:
Optional but you could probably save a layer of indentation by only including h2 and above headings

Reply:
I'm actually using a plugin for this, I'll see if it's configurable.

Reply:
If using markdown all in one then see `Markdown › Extension › Toc: Levels` and set e.g. `2..5`

Reply:
Done.